src/Synapse/OAuth2/ServerServiceProvider.php

```diff
@@ -60,13 +61,20 @@ public function setup(Application $app)
         });
 
         $app['oauth.controller'] = $app->share(function ($app) {
+            try {
+                $loginConfiguration = $app['config']->load('login');
+            } catch (Exception $e) {
+                $loginConfiguration = [];
+            }
```

Comment:
Rather than letting it error out, would it be better to just require all projects to have the `login` config?

This is the case for all other configs.